clients/egui/src/account/modals/mod.rs

```diff
@@ -0,0 +1,156 @@
+mod confirm_delete;
+mod error;
+mod help;
+mod new_file;
+mod search;
+mod settings;
+
+pub use confirm_delete::ConfirmDeleteModal;
+pub use error::ErrorModal;
+pub use help::HelpModal;
+pub use new_file::{NewFileModal, NewFileParams};
+pub use search::SearchModal;
+pub use settings::SettingsModal;
+
+use eframe::egui;
+
+#[derive(Default)]
+pub struct Modals {
```

Comment:
I think "model" might be the appropriate spelling for this and the other instances using this word.

Reply:
You also used the spelling "model" for a file.

Reply:
Nope, it's [modal](https://www.quora.com/Whats-the-difference-between-a-modal-and-dialog) haha. If it's confusing enough though I can switch back to dialog, but modal is more appropriate imo. In the GTK client, all of these dialogs are `modal` (a GTK property) for the very reason that they present content that must be addressed on its own while disallowing interaction with the main layer of the application.

I used the word "model" for miscellaneous data modelling, like we do in core and other places.

Reply:
Oh, I didn't realize you were using it in that sense. Fair enough.